wrapper/tests/test_diagnostics.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+import os
+import shield.wrapper
+import pace.util
+from mpi4py import MPI
+
+from util import get_default_config, main
+
+test_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+class DiagnosticTests(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.mpi_comm = MPI.COMM_WORLD
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        self.mpi_comm.barrier()
+
+    def test_get_diag_info(self):
+        output = shield.wrapper._get_diagnostic_info()
+        assert len(output) > 0
+        for index, item in output.items():
+            self.assertIsInstance(item.axes, int)
+            self.assertIsInstance(item.module_name, str)
+
+            self.assertIsInstance(item.name, str)
+            self.assertIsNot(item.name, "")
+
+            self.assertIsInstance(item.description, str)
+            self.assertIsInstance(item.unit, str)
+
+    def test_get_diagnostic_data(self):
+        names_to_get = ["dt3dt_1", "u10m"]
```

Comment:
While getting the accumulated diagnostic `"dt3dt_1"` is fine for testing, it is probably in better taste to illustrate getting a non-accumulated 3D diagnostic:
```suggestion
        names_to_get = ["dkt_pbl", "u10m"]
```

Reply:
This accidentally got lost in the rebase, so I re-made this change in 5bdff97.